packages/dev/core/src/Bones/skeleton.ts

```diff
@@ -189,6 +189,10 @@ export class Skeleton implements IAnimatable {
             this.prepare();
         }
 
+        if (this._isDirty) {

```

Comment:
nit: you could have merged this `if` with the previous one

Reply:
yes not sure why I did not...